### PR TITLE
Refactor GraphQL DataFetchers

### PR DESCRIPTION
**Pull request by ymorimo**
This PR is kind of big, but the points are as follows:

* `DataFetcherHelper` has been removed and the static utility methods are moved to the `DataFetcherUtils` class.
* Data fetchers for all mutation operations (except for abort operation) have been merged into one class named `MutationDataFetcherFactory` to share common logic used in the put and delete operations. This implements the [`DataFetcherFactory`](https://javadoc.io/static/com.graphql-java/graphql-java/17.2/graphql/schema/DataFetcherFactory.html) interface to provide data fetchers for each operation (put, delete, bulkPut, bulkDelete, and mutate).

### CODE REVIEW

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: graphql/src/main/java/com/scalar/db/graphql/datafetcher/MutationDataFetcherFactory.java [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+    dataFetcherForBulkPut = this::dataFetcherForBulkPut;
+    dataFetcherForBulkDelete = this::dataFetcherForBulkDelete;
+    dataFetcherForMutate = this::dataFetcherForMutate;
+  }
+
+  @Override
+  public DataFetcher<DataFetcherResult<Boolean>> get(DataFetcherFactoryEnvironment environment) {
+    GraphQLFieldDefinition fieldDefinition = environment.getFieldDefinition();
+    if (fieldDefinition.equals(tableGraphQlModel.getMutationPutField())) {
+      return dataFetcherForPut;
```
Comment (feeblefakie) [inline graphql/src/main/java/com/scalar/db/graphql/datafetcher/MutationDataFetcherFactory.java:66]: Just a question.
It's a Java functional thing and wondering how `DataFetcherResult` is converted to  `DataFetcher<DataFetcherResult>`.
It's hard to parse only from the code.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
Left a question but it doesn't block it from being merged.)
